src/components/grapher/GrapherField.vue

```diff
@@ -124,21 +124,38 @@ export default Vue.extend({
         this.$store.getters.getFrontHashOffsetY
       );
     },
+    // fourStepGridOffsetsX(): number[] {
+    //   const offset: number = this.$store.state.yardlines ? 8 : 4;
+    //   const retVal: number[] = [4, 8, this.fieldWidth - 8, this.fieldWidth - 4];
+    //   for (
+    //     let offsetX = 12;
+    //     offsetX <= this.fieldWidth - 12;
+    //     offsetX += offset
+    //   ) {
+    //     retVal.push(offsetX);
+    //   }
+    //   return retVal;
+    // },
     fourStepGridOffsetsX(): number[] {
-      const offset: number = this.$store.state.yardlines ? 8 : 4;
-      const retVal: number[] = [4, 8, this.fieldWidth - 8, this.fieldWidth - 4];
+      const retVal: number[] = [];
+      const gridSize: number = this.$store.state.gridSize;
       for (
-        let offsetX = 12;
-        offsetX <= this.fieldWidth - 12;
-        offsetX += offset
+        let offsetX = gridSize * 2;
+        offsetX < this.fieldWidth;
+        offsetX += gridSize * 2
```

Comment:
You don't have to multiply by 2 here, the X/Y scale should be 1 spacing unit to 1 step. Notice in the screenshots that the 4 step grid is actually an 8 step grid (there should be a vertical line in between each line yard, since each line yard is 8 steps between each other).